modules/installation-machines-bare-metal.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+
+[id="installation-machines-bare-metal-{context}"]
+= Creating the cluster
+
+To create the {product-title} cluster, you provision machines by using the Ingition config
+files that you generated with the installation program.
+
+.Prerequisites
+
+* Your machines must have internet access.
+* Create the required infrastructure for the cluster.
+* You obtained the installation program and generated the Ignition config files
+for your cluster.
+
+.Procedure
+
+. Obtain the {op-system-first} images.
+
+. Use the the ignition config files that you generated by using the installation
+program to create the required bootstrap, worker, and master nodes.
+
+. Monitor for bootstrap-complete
+to monitor cluster bootstrapping.
++
+----
+$ openshift-install --dir test-bare-metal upi bootstrap-complete
+INFO Waiting up to 30m0s for the Kubernetes API at https://api.test.example.com:6443...
+INFO API v1.12.4+c53f462 up
+INFO Waiting up to 30m0s for the bootstrap-complete event...
+----
++
+The command succeeds when the Kubernetes APIServer signals that it has been
+bootstrapped on the control plane machines.
+
+. Confirm that the cluster recognizes the machines:
++
+----
+$ oc get nodes
+
+NAME      STATUS    ROLES   AGE  VERSION
+master-0  NotReady  master  50s  v1.12.4+c53f462
+master-1  NotReady  master  63s  v1.12.4+c53f462
+master-2  NotReady  master  64s  v1.12.4+c53f462
+worker-0  NotReady  worker  76s  v1.12.4+c53f462
+worker-1  NotReady  worker  70s  v1.12.4+c53f462
+worker-2  NotReady  worker  69s   v1.12.4+c53f462
+----
++
+The output lists all of the machines that you created.
+
+. Approve all the pending certificate signing requests:
++
+----
+$ oc get csr -ojson | jq -r '.items[] | select(.status == {} ) | .metadata.name' | xargs oc adm certificate approve
+----
+
+. Watch the cluster components come online:
++
+----
+$ watch -n5 oc get clusteroperators
+----
+
+. Provide storage to the image registry. For non-production clusters, you
```

Comment:
@abhinavdahiya, are you aware of any specific restrictions or requirements for the image registry storage for production clusters?

Reply:
Not really. there aren't specific docs [here](https://github.com/openshift/cluster-image-registry-operator)
from [OWNERS](https://github.com/openshift/cluster-image-registry-operator/blob/9e0313ec13956395d35e1ef19fb879ab9043fd87/OWNERS#L8) @bparees can you provide some insight.

Reply:
/cc @adambkaplan @dmage 

for production usage the only supported options are either:
1) S3
2) PVC where your PVC is backed by a RWX PV on a fileserver/service that you deem production ready (ie sufficiently performant, redundant, and backed up)